services/common/src/main/java/com/linkedin/openhouse/common/stats/model/IcebergTableStats.java

```diff
@@ -36,5 +36,21 @@ public class IcebergTableStats extends BaseTableMetadata {
 
   private Long numReferencedManifestLists;
 
+  private Long numPositionDeleteFiles;
+
+  private Long totalPositionDeleteFileSizeInBytes;
+
+  private Long numEqualityDeleteFiles;
+
+  private Long totalEqualityDeleteFileSizeInBytes;
+
+  private Long numCurrentSnapshotPositionDeleteFiles;
+
+  private Long totalCurrentSnapshotPositionDeleteFileSizeInBytes;
+
+  private Long numCurrentSnapshotEqualityDeleteFiles;
+
+  private Long totalCurrentSnapshotEqualityDeleteFileSizeInBytes;
+
```

Comment:
Let's make sure these changes are safe. cc @maluchari

Reply:
@maluchari as I read in the internal wrapper,

there is a coded set of assignments from IcebergTableStats 's  attributes -> the internal event schema.

so without changing anything in the internal wrapper, the fields in IcebergTableStats go unused. 

thus it is safe to merge this PR, and care must be taken in the internal wrapper when we update the set of assignemnts and change the event schema.

Reply:
This specific block of lines LGTM